buildenv/jenkins/testJobTemplate

```diff
@@ -136,7 +138,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the test command")
 							stringParam('JVM_OPTIONS', "", "Use this to replace the test original command line options")
 							stringParam('ITERATIONS',"1", "Number of times to repeat execution of test target")
-							stringParam('LABEL', "", "Jenkins node label to run on, leave this blank to get sent to any machine matching the platform, set to node name for runs a particular machine")
+							stringParam('LABEL', LABEL, "Jenkins node label to run on, leave this blank to get sent to any machine matching the platform, set to node name for runs a particular machine")
 							stringParam('JCK_GIT_REPO', JCK_GIT_REPO, "For JCK test only")
```

Comment:
Instead of using existing LABEL I would suggest to add a new parameter EXTRA_LABEL. Hence if there is some other specific requirement it can just be passed in.

Reply:
Not a bad idea, it would make the logic more simple and more easy for the end user.

Reply:
@sophia-guo - sure. I'll make that change!